flint.ui/src/components/GCBM/Configurations/Pools.vue

```diff
@@ -1,27 +1,39 @@
 <template>
   <div class="py-4 px-8 text-earth">
-    <a-typography-title><span class="font-normal text-earth"> Configure Pools </span></a-typography-title>
+    <a-typography-title
+      ><span class="font-normal text-earth">
+        Configure Pools
+        <a-input-search
```

Comment:
can you check if the input's position is being rendered correctly? As in the [deployed url](https://gcbm-editor-65lmbieek-moja-global.vercel.app/gcbm/configurations/pools) it's being shown in the middle:
![image](https://user-images.githubusercontent.com/35102959/175779884-9c41b8be-3a63-47a2-94a5-7099fe6567b6.png)